rfc-report.jq

```diff
@@ -7,13 +7,14 @@ def has_label(l): .labels | any(.name == l);
   | length as $length
   | sort_by(.number)
   | [
-    { title: "Draft RFCs", items: map(select(has_label("status: draft"))) },
-    { title: "Unlabelled and New RFCs", items: map(select(isempty(.labels | .[]) or has_label("status: new"))) },
-    { title: "RFCs Open for Nominations", items: map(select(has_label("status: open for nominations"))) },
-    { title: "RFCs in Discussion", items: map(select(has_label("status: in discussion"))) },
-    { title: "RFCs in FCP", items: map(select(has_label("status: FCP"))) },
-    { title: "Accepted/Rejected/Closed", items: map(select(.closed_at > $closed_since)) }
-  ]
+    { title: "Draft RFCs", items: map(select(.draft)) }
+  ] + (map(select(.draft | not)) | [
+      { title: "Unlabelled and New RFCs", items: map(select(isempty(.labels | .[]) or has_label("status: new"))) },
+      { title: "RFCs Open for Nominations", items: map(select(has_label("status: open for nominations"))) },
+      { title: "RFCs in Discussion", items: map(select(has_label("status: in discussion"))) },
+      { title: "RFCs in FCP", items: map(select(has_label("status: FCP"))) },
+      { title: "Accepted/Rejected/Closed", items: map(select(.closed_at > $closed_since)) }
+  ])
```

Comment:
I'm not a huge fan of this because it evaluates each one separately. So there is nothing preventing PRs from being dropped or duplicated. For example IIUC PRs that have the label `status: draft` but aren't actually a draft will be ignored forever.

A better way to do this may be to assign a "title" to each PR then group by that field. We can then have a conditional which provides a label or has a default "unsorted" or something that makes it obvious that it needs attention.

Also I'm wondering if it makes sense to use a "real" programming language to make this easier, but I think the suggestion above can be done in jq if you bash it hard enough.